### ISSUE TITLE
simple way to set user name and add links

### ISSUE THREAD

**Issue by guaka** [guaka/hitchticker#5]
- [x] username
- [ ] and possibly links to hitchwiki profile, blog, etc.

**Comment by UserStefan** [comment]
You are talking about the style of the profile behind the name?

**Comment by guaka** [comment]
- name for indicating who posted something (which will be link to profile page)
- URLs for profile page

**Comment by UserStefan** [comment]
Yes, i understand. We can have one link to Homepage/Blog and one link to hitchwiki profile. So people are explizitly asked to use their hitchwiki profile, if they use the ticker. :)

To keep it simple i don´t think we need more. If you think it makes sense, add one field for some personal text, like in trustroots under the username.

Later we can add facebook, twitter etc. but imho this seems a bit unsexy.

**Comment by platschi** [comment]
How about OpenID login with existing Hitchwiki account? Is this technically possible?

**Comment by guaka** [comment]
now possible to add/change name/phone

idea: little profile field that is rendered as markdown (this could catch the URL stuff)

**Comment by guaka** [comment]
user name works fine, the URLs are not essential for the first race I think

**Comment by simison** [comment]
BTW people don't really have "Hitchwiki profiles", they have accounts. Well you guys do, but majority has just empty user page.
